src/begrunnelser/art16_1_anmodning_uten_art12_engelsk.js

```diff
@@ -0,0 +1,40 @@
+const art16_1_anmodning_uten_art12_engelsk = [
+  {
+    kode: 'KORTVARIG_PERIODE_RETUR_NORSK_AG',
+    term: 'The person will return to her/his Norwegian employer after a brief work period in your country.',
+  },
+  {
+    kode: 'IDEELL_ORGANISASJON_IKKE_VESENTLIG_VIRK',
+    term: 'Working for a nonprofit organization.',
+  },
+  {
+    kode: 'HJEMMEKONTOR_MEDFOELGENDE',
+    term: 'The person is accompanying her/his spouse or partner who is posted to your country, and will be working in a home office for her/his Norwegian employer.',
+  },
+  {
+    kode: 'HJEMMEKONTOR_BEGRENSET_PERIODE',
+    term: 'Will briefly be working in a home office in your country for her/his Norwegian employer.',
+  },  {
+    kode: 'DELTIDSARBEID_I_UTLANDET_MOTTAR_AAP',
+    term: 'The person receives work assessment allowance from Norway and will work part time abroad as part of the rehabilitation plan.',
+  },
+  {
+    kode: 'DEN_NORSKE_SKOLEN',
+    term: 'Working in your country for the Norwegian school, which is financially supported by the Norwegian Government.',
```

Comment:
The Norwegian School, siden det er egennavn, f.eks. https://en.wikipedia.org/wiki/The_Norwegian_School_in_London